scripts/docker/build_on_alpine_linux_3_15.Dockerfile

```diff
@@ -34,10 +35,19 @@ RUN echo '@edge-testing https://dl-cdn.alpinelinux.org/alpine/edge/testing' >> /
             pegtl@edge-testing \
             pkgconf \
             polkit-dev \
-            protobuf-dev
+            protobuf-dev \
+            tar
 ADD usbguard.tar usbguard/
 ADD catch.tar usbguard/src/ThirdParty/Catch/
 WORKDIR usbguard
 RUN git init &>/dev/null && ./autogen.sh
 RUN ./configure --with-bundled-catch || ! cat config.log
-RUN make V=1 "-j$(nproc)"
+RUN make dist
+RUN tar --version
+RUN tar xf usbguard-*.tar.gz
+RUN mv -v usbguard-*.*.*/ usbguard-release/
+RUN mkdir usbguard-release/build/
+WORKDIR usbguard-release/build/
+RUN A2X=a2x ../configure --with-bundled-catch || ! cat config.log
```

Comment:
What is the motivation behind ```A2X=a2x```? I think that's the default as well or am I wrong?